Makefile.work

```diff
@@ -312,6 +312,13 @@ endif
 ifneq ($(SECURE_UPGRADE_DEV_SIGNING_CERT),)
     DOCKER_RUN += -v $(SECURE_UPGRADE_DEV_SIGNING_CERT):$(SECURE_UPGRADE_DEV_SIGNING_CERT):ro
 endif
+# Mount the Signing prod tool in the slave container
+$(info "SECURE_UPGRADE_PROD_SIGNING_TOOL": "$(SECURE_UPGRADE_PROD_SIGNING_TOOL)")
+ifneq ($(SECURE_UPGRADE_PROD_SIGNING_TOOL),)
+	SECURE_UPGRADE_PROD_SIGNING_TOOL_DST = /sonic/scripts/$(shell basename -- $(SECURE_UPGRADE_PROD_SIGNING_TOOL))
+	DOCKER_RUN += -v $(SECURE_UPGRADE_PROD_SIGNING_TOOL):$(SECURE_UPGRADE_PROD_SIGNING_TOOL_DST):ro
+	SECURE_UPGRADE_PROD_SIGNING_TOOL := $(SECURE_UPGRADE_PROD_SIGNING_TOOL_SOURCE)
```

Comment:
This is not correct I think. from where SECURE_UPGRADE_PROD_SIGNING_TOOL_SOURCE is coming from?

The tool path is pointed by SECURE_UPGRADE_PROD_SIGNING_TOOL_DST.
Anyway in my opinion you don't need to alter the path  from vendor-specific to /sonic/script
Mount them as it is
Ex:
DOCKER_RUN += -v $(SECURE_UPGRADE_PROD_SIGNING_TOOL):$(SECURE_UPGRADE_PROD_SIGNING_TOOL):ro

Reply:
The SECURE_UPGRADE_PROD_SIGNING_TOOL when not used should be empty. But in rules/config the default to none. The "none" is itself some value and hence make 
ifneq ($(SECURE_UPGRADE_PROD_SIGNING_TOOL),) instruction wont work

I think this supposed to be
ifneq ($(SECURE_UPGRADE_PROD_SIGNING_TOOL),none)

Reply:
agree, I will add this condition

Reply:
Ack